pkg/deploys/apicurio/deployer.go

```diff
@@ -17,7 +17,7 @@ func NewDeployer() *ApiCurioDeployer {
 }
 
 func (ac *ApiCurioDeployer) GetCatalogEntries() []*brokerapi.Service {
```

Comment:
Should we also update the naming of the objects here so there's no confusion later?

Reply:
I thought about doing it but didn't want to risk breaking the installer since we are only interested in the updated plan name.